plugins/newsdownloader.koplugin/main.lua

```diff
@@ -245,11 +246,15 @@ function NewsDownloader:processAtom(feeds, limit)
         if limit ~= 0 and index - 1 == limit then
             break
         end
-        self:downloadFeed(feed, feed_output_dir)
+        if createNewsFromDescription then
+           self:createFromDescription(feed, feed_output_dir)
```

Comment:
indentation

Also this can't work. It's not `feed.description` but `feed.content` for Atom unless I'm mistaken.